test/unittest/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ project(syclblas_unittest)
 
 set(SYCLBLAS_UNITTEST ${CMAKE_CURRENT_SOURCE_DIR})
 
-include_directories(${SYCLBLAS_TEST} ${BLAS_INCLUDE_DIRS})
+include_directories(${SYCLBLAS_TEST})
```

Comment:
Is this related to this PR?

Reply:
Not strictly. Just noticed that this include was not necessary, and that ${BLAS_INCLUDE_DIRS} was always empty for me, so I removed it. Should I add it back ?

Reply:
yes, it is used when you install the system blas in a custom place